pilot/cmd/pilot-agent/main.go

```diff
@@ -84,6 +84,7 @@ var (
 	podNameVar           = env.RegisterStringVar("POD_NAME", "", "")
 	podNamespaceVar      = env.RegisterStringVar("POD_NAMESPACE", "", "")
 	kubeAppProberNameVar = env.RegisterStringVar(status.KubeAppProberEnvName, "", "")
+	serviceAccountVar    = env.RegisterStringVar("SERVICE_ACCOUNT", "", "Name of service account")
```

Comment:
Which service account ? Of Istio ?

Reply:
This is the service account corresponding to the workload pod. We need this to infer the SPIFEE identity of the workload when we generate the CSR request